docs/tutorial/avax-nft.md

```diff
@@ -0,0 +1,399 @@
+---
+title: Mint NFTs on Avalanche 🚧
+description: Learn how to start minting NFTs on Avalanche.
+issueUrl: https://github.com/protocol/nft-website/issues/224
+---
+
+# Mint NFTs on Avalanche
+
+This short tutorial will quickly guide you through getting started with an EVM-compatible NFT minting work flow on [Avalanche](https://www.avax.network/)using Node.js REPL. This is quite useful if you are migrating from Ethereum or other EVM-compatible blockchain and wish to reuse your code.
+
+## Avalanche vs Ethereum
+
+Unlike Ethereum, Avalanche is a network of multi-blockchains, one of which runs a fork of Ethereum Virtual Machine (EVM) and compatible with Ethereum.
+
+Avalanche consists of 3 subnets, X-chain, P-chain, and C-chain.
+
+- **X-chain**: Deals with exchanges of value and runs AVM (namespaced `avm`)
+- **P-chain**: Deals with platform/protocol (core) and able to create new arbitrary blockchains (namespaced `platform`)
+- **C-chain**: the EVM-compatible chain capable of running Solidity smart contract / dapps. It has Ethereum-compatible addresses (hexadecimal strings prefixed with "0x" concatenated with the rightmost 20 bytes of the Keccak-256 hash ECDSA public key)
+
+Most beginner's confusion will be from distinguishing these different subnets. Note that only the C-chain is EVM-compatible and has Ethereum-compatible addresses. and most of dapps will be interacting with this chain.
+
+![Avalanche's diagram of subnets](https://docs.avax.network/assets/images/image(21)-3c5cb7f1f21926b05ae3631f453ed49d.png)
+
+## Setting up
+
+The quickest way to start is to run a group of simulator nodes locally. To do that, follow these steps:
+
+### Install and download
+
+- [Install Go](https://go.dev/dl/). Make sure to set `$GOPATH` variable to where you keep Go code (i.e. `$HOME/go`).
+
+- [Clone Avalanchego](https://github.com/ava-labs/avalanchego) (Avalanche node) and [Avalanch local simulator](https://github.com/ava-labs/ava-sim#readme). Make sure they are inside `$GOPATH/src/github.com/ava-labs`.
+
+- Make sure you have [Node.js](https://nodejs.org/en/) on your system to create the NFT app.
+
+### Run local simulator nodes
+
+- Build both projects with `./scripts/build.sh` included in the downloaded repositories.
+
+- In `ava-sim`, run the simulator with `./scripts/run.sh`, which relies on `avalanchego` being in the right place. The simulator runs a local network of 5 nodes listening on different ports. We will be using a node listening on port 9650.
+
+### Create a keystore user and add test fund
+
+Create a keystore user and store the credential on the target node (here, the node running on port 3650). Send a request to this API endpoint:
```

Comment:
*store your credentials

Reply:
I back this as well. 

Create a new keystore user requiring to create a new userName and password, maybe mentioned this earlier than later. I was a bit confused when I saw the line below the code block because I was wondering where do I get myUsername and myPassword.

Reply:
Thanks!